SEC/Equities/EquityInstruments.rdf

```diff
@@ -832,12 +947,12 @@
 		<fibo-fnd-utl-av:explanatoryNote xml:lang="en">A relatively small float results in higher volatility, as a large purchase or sell order will have significant influence on the value of the stock.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:DatatypeProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-sec-eq-eq;hasGracePeriod">
-		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-dt-fd;hasDuration"/>
-		<rdfs:label>has grace period</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-sec-eq-eq;PreferredShareCoupon"/>
-		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;Duration"/>
-		<skos:definition>indicates a period of time after a coupon payment becomes due, before the issuer is subject to penalties</skos:definition>
+	<owl:ObjectProperty rdf:about="&fibo-sec-eq-eq;hasMinimumRedemptionPrice">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-acc-cur;hasPrice"/>
+		<rdfs:label>has minimum redemption price</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-sec-eq-eq;EquityRedemptionProvision"/>
+		<rdfs:range rdf:resource="&fibo-fbc-fi-ip;MarketPrice"/>
```

Comment:
As per earlier comment MarketPrice is wrong here

Reply:
Ok.  While we think market price is appropriate for some cases, we've loosened it to monetary price.  There is likely a requirement for the more constrained market price from a provenance perspective for evidentiary purposes, but it's possible that a given contract won't require that.